src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -7015,6 +7014,11 @@ static void sofia_handle_sip_r_invite(switch_core_session_t *session, int status
 		int has_t38 = 0;
 		const char *drrrf_chanvar_str = switch_channel_get_variable(channel, "disable_recovery_record_route_fixup"); // disable_recovery_record_route_fixup as a chanvar
 		switch_bool_t drrrf_chanvar_zstr = zstr(drrrf_chanvar_str), drrrf_chanvar = switch_true(drrrf_chanvar_str);
+		switch_bool_t handle_update = switch_channel_var_true(channel, "handle_update");
+
+		if (status > 180 && (handle_update || sofia_test_pflag(profile, PFLAG_HANDLE_UPDATE))) {
```

Comment:
What if I set `handle_update=false`? This should also disable this feature even if its enable in profile variable